### ISSUE TITLE
Making ComposePanel transparent

### ISSUE THREAD

**Issue by hrach** [JetBrains/compose-multiplatform#147]
I'd like to show content under ComposePanel, so I would like to make the default background transparent. Is there a way to make ComposePanel transparent? Thank you for the great work.

**Comment by Rsedaikin** [comment]
Hi! I am already working on several prototypes that support transparency, but currently the ComposePanel background is filled with white by default.

**Comment by graycatdeveloper** [comment]
@Rsedaikin Is there any progress?

**Comment by supersnek** [comment]
As a temporary workaround, before creating `ComposePanel` you can add `UIManager.put("Panel.background", java.awt.Color(0, 0, 0, 0))`  and it will change background to transparent

**Comment by wakaztahir** [comment]
I've put this property UIManager.put("Panel.background", java.awt.Color(0, 0, 0, 0))

but panel background is still white !

**Comment by zwz-coding** [comment]
Is there a solution now?

**Comment by GaborPeto** [comment]
Are there any updates?

**Comment by MatkovIvan** [comment]
Offscreen rendering allows that. You can enable it via `compose.swing.render.on.graphics` system property - it's available since 1.5.0. See https://blog.jetbrains.com/kotlin/2023/08/compose-multiplatform-1-5-0-release/#enhanced-swing-interop

<img width="146" alt="Screenshot 2024-02-28 at 09 23 00" src="https://github.com/JetBrains/compose-multiplatform/assets/1836384/bf1b9ce5-69a2-4e12-8a5f-081857a6f6f5">

```kt
fun main() = SwingUtilities.invokeLater {
    System.setProperty("compose.swing.render.on.graphics", "true")

    val window = JFrame()
    window.defaultCloseOperation = WindowConstants.EXIT_ON_CLOSE
    window.contentPane.layout = null
    window.contentPane.add(ComposePanel().apply {
        setContent {
            Box(Modifier.fillMaxSize().background(Color.Green.copy(alpha = 0.5f)))
        }
        setBounds(20, 20, 200, 200)
    })
    window.contentPane.add(JButton("Button").apply {
        setBounds(10, 10, 100, 50)
    })
    window.setSize(800, 600)
    window.isVisible = true
}
```

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5587">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
